pygmt/src/sphinterpolate.py

```diff
@@ -0,0 +1,70 @@
+"""
+sphinterpolate - Spherical gridding in tension of data on a sphere
+"""
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import (
+    GMTTempFile,
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+from pygmt.io import load_dataarray
+
+
+@fmt_docstring
+@use_alias(
+    G="outgrid",
+    I="spacing",
+    R="region",
+    V="verbose",
+)
+@kwargs_to_strings(I="sequence", R="sequence")
+def sphinterpolate(table, **kwargs):
```

Comment:
Change to using `data` instead of `table` as per https://github.com/GenericMappingTools/pygmt/issues/1479#issuecomment-912203679.
```suggestion
def sphinterpolate(data, **kwargs):
```